.github/workflows/build.yml

```diff
@@ -111,13 +111,19 @@ jobs:
         cmake --build . --config Release
         cmake --install . --config Release
 
-    - name: Build pc-ble-driver-py wheels
-      if: runner.os != 'Windows'
+    - name: Build pc-ble-driver-py wheels (Linux/MacOS)
       working-directory: pc-ble-driver-py
+      if: runner.os != 'Windows'
       run: |
-        python3 setup.py bdist_wheel --build-type Release
-
-    - name: Build pc-ble-driver-py wheels windows
+        if [ "$RUNNER_OS" == "macOS" ]; then
+          python3 setup.py bdist_wheel --build-type Release --py-limited-api=
+        elif [ "$RUNNER_OS" == "Linux" ]; then
+          python3 setup.py bdist_wheel --build-type Release
+        else
+          echo "Unsupported OS: $RUNNER_OS"
+          exit 1
+        fi
+    - name: Build pc-ble-driver-py wheels (Windows)
```

Comment:
Clearer to add own step for all OS's or include all to same step.

Reply:
62bc1c79799141265ad9fd8258213584c3956688